ios/BT/API/Model/CheckInStatus.swift

```diff
@@ -0,0 +1,39 @@
+import ExposureNotification
+import Foundation
+import RealmSwift
+
+@objcMembers
+class CheckInStatus: Object, Codable {
+  @objc dynamic var id: String = .default
+  @objc dynamic var date: Int = 0
+  var feelingGood: Bool = false
+  var symptoms: List<String> = List<String>()
+
+  init(date: Int,
+       feelingGood: Bool,
+       symptoms: [String]) {
+    self.date = date
+    self.feelingGood = feelingGood
+    let list = List<String>()
+    list.append(objectsIn: symptoms)
+    self.symptoms = list
+    super.init()
+  }
+
+  required init() {
+    super.init()
+  }
+
+  override class func primaryKey() -> String? {
+    "date"
+  }
+
+  var asDictionary : [String: Any] {
+    return [
+      "posixDate": date,
```

Comment:
Is this the serialized structure that would be exposed? How do you feel about exposing the id? We will probably needed to render collections.

Reply:
hm, our thinking was that the posixDate would be the record's id, because no two records could exist for the same exact date, so we have a unique identifier at hand without needing to add an additional `id` property

Reply:
The requirements seem to indicate that we could have multiple symptom entries per day, so we might make use of this unique identifier more often than not.

Reply:
wouldn't the posixDate be unique because it is expressed in seconds? 🤔

Reply:
we were imagining that the date sent down from the JS layer would be a precise timestamp (i.e. not rounded to the day)